guides/doc-Managing_Hosts/topics/proc_removing-a-host-from-satellite.adoc

```diff
@@ -15,8 +15,10 @@ In both cases, {Project} removes a host completely.
 
 [WARNING]
 ====
-If a host record that is associated with a virtual machine is deleted, the virtual machine will be deleted as well.
-To avoid deleting the virtual machine in this situation, disassociate the virtual machine from {Project} without removing it from the hypervisor.
+The deletion of the virtual machine on the hypervisor depends on the setting of `Destroy associated VM on host delete` which is set to `no` by default.
```

Comment:
I have no opinion on the exact wording but since "deleting" is often used somewhat vaguely to mean "delete the foreman entity but not the thing itself", and this is precisely **not** what we mean here, we need _something_ to make the distinction unambiguous.

Reply:
Reworded hypervisor to compute resource.